.github/workflows/kubectl-az.yml

```diff
@@ -195,3 +197,40 @@ jobs:
         shell: bash
         run: |
           make integration-test -o kubectl-az
+
+  release:
+    name: Release
+    needs: [ integration-tests ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+    # Checkout the repo to get the .krew.yaml file
+    - uses: actions/checkout@v3
+    - name: Get all artifacts.
+      uses: actions/download-artifact@v3
+    - name: Rename all artifacts to kubectl-az-${{ github.ref_name }}.tar.gz
+      shell: bash
+      run: |
+        for i in kubectl-az-*-*-tar-gz/kubectl-az-*-*.tar.gz; do
+          mv $i $(dirname $i)/$(basename $i .tar.gz)-${{ github.ref_name }}.tar.gz
+        done
+    - name: Create Release
+      id: create_release
+      uses: softprops/action-gh-release@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        name: Release ${{ github.ref }}
+    - name: Upload kubectl-az binaries to the release
+      uses: csexton/release-asset-action@v2
+      with:
+        pattern: "kubectl-az-*-*-tar-gz/kubectl-az-*-*.tar.gz"
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        release-url: ${{ steps.create_release.outputs.upload_url }}
+    - name: Update new version in krew-index
+      if: github.repository == 'azure/kubectl-az'
+      uses: rajatjindal/krew-release-bot@v0.0.46
+      with:
+        workdir: /home/runner/work/kubectl-az/kubectl-az
```

Comment:
The default value here for the plugin seems to be `env.GITHUB_WORKSPACE` and that seems to be the same path for this repo as you have written.
But I prefer having the default (environment variable) here. What do you think?

Reply:
You're right. That's the default value and I tested it and it is indeed `/home/runner/work/kubectl-az/kubectl-az`. Thanks!